src/Reusables/Navbar.js

```diff
@@ -0,0 +1,33 @@
+
+function Navbar() {
+    return (
+        <header class="nav-header">
```

Comment:
use className instead of class as it would show warnings

Reply:
Done